doc/user-guide/configuration.rst

```diff
@@ -41,6 +41,21 @@ Arduino IDE
 
 2. The default configuration file, :github-blob:`src/config_default.h`.
 
+Default Settings
+^^^^^^^^^^^^^^^^
+
+In this section you can find some important and widly used settings (e.g.
+serial port configuration).
+
+Serial Port
+"""""""""""
+
+The default baudrate for ESP boards is 76800, for ESP32, SPC5, and
+Arduino DUE is 115200, and for all the other boards the baudrate is 38400.
```

Comment:
To minimize maintenance the actual baudrate values should not be written here as it's unlikely one will remember to update them if changed in the configuration file. Also, when adding a new board it's preferred to configure it for 115200 baud, not 38400. So that part of the documentation will be outdated quickly as well.

It's a good idea to present the default baudrate to the user where it's easily found, probably on each of the specific [board pages](http://simba-os.readthedocs.io/en/latest/boards.html). There is already a script generating those pages, to one could easily add the serial port (console) baudrate.

Reply:
I care about low maintenence cost and user friendlyness. Adding links to relevant information is welcome throughout the documentation.

Only for the ESP boards, as Simba should use the same default baudrate as Espressif. Other non-115200 boards were created a long time ago, before I decided to use 115200. These boards should be changed to use 115200.